core/src/main/java/io/grpc/CallOptions.java

```diff
@@ -170,6 +175,18 @@ public CallOptions withAffinity(Attributes affinity) {
   }
 
   /**
+   * Sets whether the call is not
+   * <a href="https://github.com/grpc/grpc/blob/master/doc/fail_fast.md">fail fast</a>.
+   * 'Fail fast' is the default option for gRPC calls and 'Wait for ready' is the opposite to it.
+   */
+  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/1915")
+  public CallOptions withWaitForReady(boolean waitForReady) {
```

Comment:
We talked with @ctiller and they may change C++ if there is general agreement across languages. The original thought was that some languages would do it one way and others may do it another. However, that was because "fail fast" was such a better descriptor than "ignore connectivity". "Wait for ready" actually seems good enough that we may drop "fail fast" terminology altogether across languages.